source/install/prod-windows-2012.rst

```diff
@@ -0,0 +1,419 @@
+
+   a. On the **Type and Networking** page, select the appropriate **Config Type** based on your desired usage.
+   b. On the **Accounts and Roles** page, enter a MySQL Root password and press the Next button.
+   c. On the **Windows Service** page, keep the defaults and press the Next button.
+   d. On the **Plugins and Extensions** page, keep the defaults and press the Next button.
+   e. On the **Apply Server Configuration** page, press the Execute button to configure the MySQL Server installation.
+
+10. Returning to the **Product Configuration** page, press the Next button.
+
+11. On the **Installation Complete** page, press the Finish button.
+
+Configure Windows PATH
+^^^^^^^^^^^^^^^^^^^^^^
+
+Adding MySQL to the Windows PATH variable will allow you run various commands from the console, and 
+allow 3rd party applications to execute various MySQL commands.
+
+12. Press the ``WIN + PAUSEBREAK`` key combination to open the **System** Control Panel Applet.
+
+13. Select the **Advanced System Settings** link.
+
+14. Select the **Advanced** tab. 
+
+15. Press the **Environment Variables** button.
+
+16. In the **System Variables** section, select the **Path** variable in the list, then press the Edit button.
+
+17. Append the path to your MySQL Server bin directory (make sure to add a semi-colon after any existing 
+    items in the path before appending the new path). The default path for a 64-bit 
+    installation is ``C:\Program Files\MySQL\MySQL Server 5.7\bin``.
+
+Create Mattermost Database
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now that MySQL has been installed and configured, a database and user account must be provisioned 
+for Mattermost to utilize.
+
+18. Open a command prompt and enter ``mysql -u root -p`` to connect to the MySQL server
+
+19. When prompted, enter the password created in step 9.a.       
+
+20. At the ``mysql>`` prompt, execute the following commands to create the database and user account
+
+    .. code:: sql
+
+       CREATE DATABASE mattermost;
+       CREATE USER mmuser IDENTIFIED BY 'mmuser_password';
+       GRANT ALL ON mattermost.* TO mmuser;
+       exit
+
+21. To confirm the database and user were configured correctly
+
+    a. Connect to the MySQL server/datbase by executing ``mysql -u mmuser -p mattermost`` 
+    b. When prompted, entering ``mmuser_password``
+    c. If If successful, you will be at the ``mysql>`` prompt 
+    d. Type ``exit`` to finish
+
+Set up Mattermost Server
+------------------------
+
+1. For the purposes of this guide we will assume this server has an IP
+   address of 10.0.0.2
+
+2. `Download <https://www.mattermost.org/download/>`__ the latest Mattermost Server by opening your 
+   favorite browser and navigating to 
+   ``https://releases.mattermost.com/X.X.X/mattermost-team-X.X.X-windows-amd64.tar.gz`` 
+   where `X.X.X` is the latest Mattermost release version. For example, 3.4.0.
+
```

Comment:
The link should be ``https://releases.mattermost.com/X.X.X/mattermost-team-X.X.X-windows-amd64.zip`` because the ".tar.gz" does not exist.

If you're going to mention the version, then it needs to be updated every release. And if it's going to be updated every release, then might as well code it directly into the link, so that it becomes ``https://releases.mattermost.com/3.5.0/mattermost-team-3.5.0-windows-amd64.zip``. And if you make it a live link, then people can click and download directly from the docs without having to visit the download page.